### PR TITLE
Aegun and pins

### PR DESCRIPTION
**Pull request by ZioZan**
Advanced energy gun can fire stun shots and disablers.
Advanced energy gun can be research after failing to max reliability, then it won't emit radiation again.
Pins can be removed with plasma cutter (they will be destroyed in process).
Loyalty pins removed from autolathe.
Loyalty pins in protolathe no longer require mining and their tech level requirements have been lowered.

### CODE REVIEW

**Comment by octareenroon91** [comment]
I can go for loyalty pins using the easier mineral cost, but what does it change the balance to remove them from autolathe ?

**Comment by ZioZan** [comment]
The point of this is security buff which should decrease the number people stealing guns from armory. Now security can replace normal pins in armory guns with loyalty pins. Traitors will have to loyalty implant themselves or buy guns with TC. I removed pins from autolathe because I think science should work for this upgrade, and not just get it at round start. As any upgrade, it should be earned. People can still order normal pins at cargo.

**Comment by Koriath** [comment]
ZioZan, adding loyalty pins to autolathe was done a couple of months ago.
The idea was that people who wanted to use guns, would need to implant themselves. And if they shot a sec officer with said guns, they'd earn a ban since they were breaking the rules (loyalty implanted = you don't attack Sec Officers).
This was meant to stop revenge-looting of the armory, and to easily allow Sec Officers to draft people in crysis.

We can try your solution, but this might simply make people not care enough to make the loyalty pins in the first place.
I'd simply change all Armory guns to have loyalty pins in them from roundstart.

**Comment by ZioZan** [comment]
> This was meant to stop revenge-looting of the armory

This can't be true because I added pin replacement recipe in this pull. As I remeber, autolathe loyalty pins replaced normal pins, that happened when we had too many valid-hunting scientists. So now scientists have to either implant themselves or ask for/steal normal pins. 

> I'd simply change all Armory guns to have loyalty pins in them from roundstart.

I think this is bad idea. People should decide on their own, if they want loyalty pins in armory guns, in my opinion. And stealing from armory is viable strategy for antagonists, I think it should be anticipated however. Which is why I made this pull the way it is. It does not matter much if there are pins in autolathe or not. As you can make them in protolathe now, as well as loyalty implants. I can re-add them back, if you guys want it.

File: code/modules/research/designs/weapon_designs.dm [hunk 1]
```diff
@@ -238,7 +238,7 @@
 //WT550 Mags
 
 /datum/design/mag_oldsmg
-	name = "WT-550 Auto Gun Magazine (4.6�30mm)"
+	name = "WT-550 Auto Gun Magazine (4.6×30mm)"
```
Comment (octareenroon91) [inline code/modules/research/designs/weapon_designs.dm:241]: I'm not quite sure what the change is here, maybe the x in `4.6×30mm` ?

Comment (ZioZan) [inline code/modules/research/designs/weapon_designs.dm:241]: Probably, I did not change anything in that line.

**Pull request closed without merging**